spring-boot-admin-server-ui/src/main/frontend/views/applications/index.vue

```diff
@@ -312,9 +312,14 @@ const grouped = computed(() => {
     [],
   );
 
-  return sortBy(list, [(item) => item.status]);
+  return sortBy(list, [(item) => getApplicationStatus(item)]);
 });
 
+function getApplicationStatus(item: InstancesListType): string {
+  return applicationStore.findApplicationByInstanceId(item.instances[0].id)
+    ?.status;
```

Comment:
this is always taking the status of the first instance of this application

but there may be down and up instances in this application, so I think we need to aggregate the overall status of the application by going through all instances and taking the "worst" status

Reply:
We're just trying to find the application by it's first instanceId here. The status is then taken from the application itself.